### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR Description
Upgrade dependencies.
-- Upgrade Vertx to 4.3.8
-- Update owasp suppression
-- Update owasp gradle plugin
## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # Changelog
 
+## Next release
+### Features Added
+-
```
Comment (siladu) [inline CHANGELOG.md:5]: nit: empty bullets here and below, maybe was intentional though :)

File: gradle/owasp-suppression.xml [hunk 2]
```diff
@@ -1,25 +1,10 @@
...
-    <suppress until="2023-02-15">
+    <suppress until="2023-03-15">
         <notes><![CDATA[
-            Temporarily suppress CVE-2022-38752. Reporting as false positive https://github.com/OSSIndex/vulns/issues/328
-          More info on the fix here https://bitbucket.org/snakeyaml/snakeyaml/issues/531/stackoverflow-oss-fuzz-47081
+            Temporarily suppress CVE-2022-1471 till the fix is released. Web3Signer only parses specified yaml files.
         ]]></notes>
         <cve>CVE-2022-1471</cve>
     </suppress>
-    <suppress until="2023-02-15">
```
Comment (siladu) [inline gradle/owasp-suppression.xml:11]: yay, feels good to remove these 👍

File: gradle/owasp-suppression.xml [hunk 3]
```diff
@@ -1,25 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <!-- See https://jeremylong.github.io/DependencyCheck/general/suppression.html for examples -->
-    <suppress until="2023-02-15">
+    <suppress until="2023-03-15">
         <notes><![CDATA[
-            Temporarily suppress CVE-2022-38752. Reporting as false positive https://github.com/OSSIndex/vulns/issues/328
-          More info on the fix here https://bitbucket.org/snakeyaml/snakeyaml/issues/531/stackoverflow-oss-fuzz-47081
+            Temporarily suppress CVE-2022-1471 till the fix is released. Web3Signer only parses specified yaml files.
```
Comment (siladu) [inline gradle/owasp-suppression.xml:6]: nit: Is there a link to track it?

**Review by siladu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (usmansaleem) [reply CHANGELOG.md:5]: intentional :D ...

File: gradle/owasp-suppression.xml (hunk 3 above)
Comment (usmansaleem) [reply gradle/owasp-suppression.xml:6]: good point, updated the the bitbucket link.

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**